Expensify/App
# Add manual address to debitCardPage

### Details
Customers have no way to enter their address if they can't find it in our Address Search component. This is a temporary fix while we are working on a better solution for it. It follows the same pattern found [here](https://github.com/Expensify/App/blob/main/src/pages/ReimbursementAccount/CompanyStep.js#L229-L282).

### Fixed Issues
$ https://github.com/Expensify/App/issues/6794

### Tests
1. Log into the app.
2. Navigate to `Profile > Payments > Add Payment method > Debit card`
3. Verify that clicking `Can't find your address? Enter it manually` displays the `City`, `State` and `Zip code` fields.
4. Do the same for `/bank-account/company`, `/bank-account/personal-information` and `/bank-account/contract`

### QA Steps
Tests above.

### Tested On


### Screenshots


#### Web

![web](https://user-images.githubusercontent.com/22219519/146847164-13e1c505-6bd4-453c-ac8b-5c55bb453d60.png)

#### Mobile Web

![mweb](https://user-images.githubusercontent.com/22219519/146847182-7206ff0a-88b1-44fe-adf9-764c7d72cd1b.png)

#### Desktop

<img width="1200" alt="desktop" src="https://user-images.githubusercontent.com/22219519/146847196-6006e475-5128-46ea-9bc6-4e86d7557251.png">

#### iOS

![ios](https://user-images.githubusercontent.com/22219519/146847206-9a663070-683e-4c08-90c9-5db91f4a1f8c.png)

#### Android

![android](https://user-images.githubusercontent.com/22219519/146847260-7b8854b0-1dd3-4c71-9bd7-05647b109245.png)
